packages/flutter/lib/src/gestures/tap.dart

```diff
@@ -61,26 +91,73 @@ typedef void GestureTapCancelCallback();
 /// pointer interactions during a tap sequence are not recognized as additional
 /// taps. For example, down-1, down-2, up-1, up-2 produces only one tap on up-1.
 ///
+/// The lifecycle of events for a tap gesture are as follows:
+///
+/// * [onTapDown], which triggers after a short timeout ([deadline]) even if the
+///   gesture has not won its arena yet.
```

Comment:
You might explain what an arena is, or link to an appropriate class. It's not that clear what is meant by arena if this is the first gesture class you encounter.

Reply:
done